src/mme-app/contextManager/subsDataGroupManager.cpp

```diff
@@ -74,6 +78,8 @@ namespace mme
 		MmeTauProcedureCtxtManagerm_p = new MmeTauProcedureCtxtManager(8000);
 		S1HandoverProcedureContextManagerm_p = new S1HandoverProcedureContextManager(8000);
 		MmeErabModIndProcedureCtxtManagerm_p = new MmeErabModIndProcedureCtxtManager(8000);
+		MmeSmCreateBearerProcCtxtManagerm_p = new MmeSmCreateBearerProcCtxtManager(8000);
```

Comment:
please check exist new code...i updated some of the pool sizes to 16k

Reply:
Done.